packages/historical-sync/src/index.ts

```
@@ -1,7 +1,7 @@
-import { fetchPoolSnapshot } from './fetchers';
-import { storePoolSnapshots } from './services';
-import { PoolSnapshotRow } from './types';
-import { getHourlyBlocks } from './utils/helpers';
+import { fetchPoolSnapshot } from '@pokt.metrics/indexer/dist/fetchers';
```

ESLint

[error] Resolve error: typescript with invalid interface loaded as resolver

(import/namespace)